packages/f-checkout/src/components/Checkout.vue

```diff
@@ -328,6 +328,14 @@ export default {
         */
         isValidPhoneNumber () {
             return validations.isValidPhoneNumber(this.customer.mobileNumber, this.$i18n.locale);
+        },
+
+        /*
```

Comment:
Do you need to add test for this?

Reply:
Hey! There are tests in place already to make sure the validations work correctly. I've also done tests for the validations in f-services. 🙂